content/docs/securitypolicy/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Security Policy"
+linkTitle: "Security Policy"
+weight: 13
+Description: >
+  Dell Container Storage Modules (CSM) Security Policy
+---
+
+
+The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec), 
+Instructions for reporting a vulnerability can be found on the [Dell Vulnerability Policy](https://www.dell.com/support/contents/en-in/article/product-support/self-support-knowledgebase/security-antivirus/alerts-vulnerabilities/dell-vulnerability-response-policy#:~:text=To%20report%20a%20security%20vulnerability%20or%20issue%20in%20Dell.com,instructions%20to%20reproduce%20the%20issue)
```

Comment:
We are providing two ways to report vulnerabilities. One through this portal and the other through GitHub. Is this the recommended way from the PSO office?